page-template.html

```diff
@@ -3,37 +3,33 @@
   <head>
     <meta charset="UTF-8" />
     <title>Progressive Web App Workshop</title>
-    <meta
-      name="description"
-      content="A progressive web app workshop presented by Cloud Four"
-    />
+    <meta name="description" content="A progressive web app workshop presented by Cloud Four" />
```

Comment:
@grigs This is the most important file to review. My goal was to try and remove things we don't need for the workshop. 🙂